src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Why is `Lottery` extending `ColorSupplier` class? Please remove it as we don't want to inherit anything from this class.